appinventor/appengine/src/com/google/appinventor/server/project/youngandroid/YoungAndroidProjectService.java

```diff
@@ -132,9 +129,13 @@ public final class YoungAndroidProjectService extends CommonProjectService {
   // host[:port] to use for connecting to the build server
   private static final Flag<String> buildServerHost =
       Flag.createFlag("build.server.host", "localhost:9990");
+  private static final Flag<String> buildServerPassword =
+          Flag.createFlag("build.server.password", "");
   // host[:port] to use for connecting to the second build server
   private static final Flag<String> buildServerHost2 =
       Flag.createFlag("build2.server.host", "");
+  private static final Flag<String> buildServerPassword2 =
+          Flag.createFlag("build2.server.password", "");
```

Comment:
Indentation. Also, I wonder if this password feature can be generalized since we will need to add a password for the iOS build entry added in #3468

Reply:
Hmm, I'm not very familiar with iOS build system, but from that PR it seems like the password is app specific, right? Here the password would be buildserver specific, not "per app"; in theory, if a password is required for the app, it will still be passed down to the compiler.